.github/workflows/code-cleanup-report.yml

```diff
@@ -0,0 +1,95 @@
+name: Code Cleanup Report
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  unused-vars:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        package:
+        - fx-core
+        - server
+        - vscode-extension
+        - manifest
```

Comment:
why select these 4 packages? why not contains cli, api, sdk or others?

Reply:
I missed CLI and API, will add them.

The checks in this pipeline aim to detect missing part during code cleanup. For example, I deleted file A, but forgot to delete file B (or part of file B) which is only referenced by file A. For other packages, the case will not happen since we're not going to delete code in them. So excluded them to avoid distraction.
But I will check code owners for the whole repo (e.g. the templates). For same reason of avoiding distraction, the pipeline will only report code owner issues for selected packages.